protocol/syncer.go

```diff
@@ -513,9 +513,14 @@ func (s *Syncer) WatchSyncWithPeer(p *syncPeer, handler func(b *types.Block) boo
 			s.logger.Info("Connection to a peer has closed already", "id", p.peer)
 			break
 		}
+
 		b, err := p.popBlock(popTimeout)
 		if err != nil {
-			s.logger.Error("failed to pop block", "err", err)
+			if errors.Is(err, ErrPopTimeout) {
+				s.logger.Debug(fmt.Sprintf("failed to pop block within %ds", popTimeout))
```

Comment:
I think it's ok this log has info level, too

Reply:
@brkomir We reached a consensus of having the Warn level with a message saying: "failed to pop block within %ds, please check if all the validators are running"

Reply:
Also include the ID of the Peer.

Reply:
Done https://github.com/0xPolygon/polygon-sdk/pull/218/commits/a63e3b1996d3790840e45d3c8f4f203d8eecf428